src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -173,6 +173,8 @@ namespace winrt::TerminalApp::implementation
                                                           const Microsoft::Terminal::TerminalControl::PasteFromClipboardEventArgs eventArgs);
 
         void _OpenHyperlinkHandler(const IInspectable sender, const Microsoft::Terminal::TerminalControl::OpenHyperlinkEventArgs eventArgs);
+        bool _IsUriSupported(const winrt::Windows::Foundation::Uri parsedUri);
```

Comment:
nit

```suggestion
        bool _IsUriSupported(const winrt::Windows::Foundation::Uri& parsedUri);
```

(not necessary to increment the reference count by copying a pointer into this guy... but it likley doesn't matter)